app/assets/javascripts/discourse/lib/lightbox.js.es6

```diff
@@ -6,6 +6,7 @@ export default function($elem) {
   if (!$elem) {
     return;
   }
+  const original_meta = $("meta[name=viewport]");
```

Comment:
Rather than replacing the entire tag, couldn't we replace the `content` attribute?

Additionally javascript variables in our codebase should be `camelCase`.

Reply:
If I would only go about changing the content attribute of the meta tag, the selector would no long be stored in `originalMeta`. I think this would decrease performance then because it has to select `$("meta[name=viewport]")` on more time.

Reply:
I think the performance of finding a meta tag is negligible compared to the memory churn of replacing a DOM node.

Reply:
Good to know. Thanks! :)